src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3326,7 +3326,7 @@ private void CreateContainerProcessInternal()
                     // Hyper-V container (i.e., RuntimeId is not empty) uses Hyper-V socket transport.
                     // Windows Server container (i.e., RuntimeId is empty) uses named pipe transport for now.
                     // This code executes `pwsh.exe` as it exists in the container which currently is
```

Comment:
@SteveL-MSFT I must ask this :-)  If MSFT team plan to replace Windows PowerShell with the PowerShell should we replace/rename `pwsh` to `powershell` too?

Reply:
That won't happen anytime soon.  We can revisit when that time comes.